pkg/backup/request.go

```diff
@@ -10,6 +10,7 @@ import (
 // materialized (e.g. backup/snapshot locations, includes/excludes, etc.)
 type Request struct {
 	*velerov1api.Backup
+	PodVolumeBackup *velerov1api.PodVolumeBackup
```

Comment:
This needs to be a slice, as one `Backup` can have more than one `PodVolumeBackups`. I would also group the new field with the other non-embedded fields just below (i.e. leave L13 blank, move to L14).